README.md

```diff
@@ -110,6 +110,20 @@ order data.
 
  * `stereo_ordering101.ipynb`.
 
+## STAC asset service
+
+SpatioTemporal Asset Catalog (STAC) is a specification designed to establish
+a standard in geospatial data. UP42 has a STAC API set `/v2/assets/stac` that allows
+you to retrieve STAC objects mapped to your UP42 assets.
+
+### Jupyter notebook
+
+In the `notebooks` directory there is a Python
+[Jupyter](https://jupyter.org) notebook that makes use of the
+[UP42 SDK](https://sdk.up42.com/) and
```

Comment:
It uses the SDK? Then it needs to just use the SDK and get rid of the _custom_ auth altogether.

Reply:
Ditto for handling asset download, etc.